### ISSUE TITLE
peerDependencies will update to laravel-mix v4

### ISSUE THREAD

**Issue by bolechen** [MaximVanhove/laravel-mix-alias#1]

**Comment by MaximVanhove** [comment]
Dear @bolechen 

Can you elaborate more on the issue you are having?

**Comment by bolechen** [comment]
when i run `yarn check`, it toke me

```
$ yarn check
$ yarn check v1.13.0
$ error "laravel-mix-alias#laravel-mix@^2.0.0" doesn't satisfy found match of "laravel-mix@4.0.15"
```

**Comment by MaximVanhove** [comment]
Dear @bolechen 

Should be fixed in 1.0.1
